packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -242,10 +247,10 @@ class SnackBar extends StatefulWidget {
     this.dismissDirection = DismissDirection.down,
     this.clipBehavior = Clip.hardEdge,
   }) : assert(elevation == null || elevation >= 0.0),
-       assert(
-         width == null || margin == null,
-         'Width and margin can not be used together',
-       );
+        assert(
```

Comment:
You'll need to revert the autoformatting changes, here and below. 

See the 'Tip' info at https://github.com/flutter/flutter/wiki/Setting-up-the-Framework-development-environment#set-up-your-environment - the code in the framework needs to have it turned off.

Reply:
Resolved